alkira/resource_alkira_connector_akamai_prolexic_helper.go

```diff
@@ -0,0 +1,80 @@
+package alkira
+
+import (
+	"log"
+
+	"github.com/alkiranet/alkira-client-go/alkira"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func expandConnectorAkamaiTunnelIps(in *schema.Set) []alkira.ConnectorAkamaiProlexicTunnelIp {
```

Comment:
We really should have tests for these "expand" type helper functions. The NAT policy rule bug we ran into could've been avoided by just checking that fields were being read in correctly.

Reply:
I did. Every single field was tested through TF configs, NAT policy simply is simply not tested enough.